src/math/mat4.js

```diff
@@ -1270,6 +1270,24 @@ class Mat4 {
         return t;
     }
 
+    // converts this matrix to supplied Mat3 format
+    toMat3(m3) {
+        var src = this.data;
+        var dst = m3.data;
+
+        dst[0] = src[0];
+        dst[1] = src[1];
+        dst[2] = src[2];
+
+        dst[3] = src[4];
+        dst[4] = src[5];
+        dst[5] = src[6];
+
+        dst[6] = src[8];
+        dst[7] = src[9];
+        dst[8] = src[10];
+    }
```

Comment:
Not sure this quite matches the naming convention of:

```
pc.Thing#setFromOtherThing
```

e.g. `pc.Quat#setFromMat4`

I mean, we also have `pc.Quat#toString` say. Ugh, I hate these sorts of consistency problems....

Reply:
ok, I'll change it to Mat3.setFrom(mat4) .. that seems more consistent?
probably easier to read as well, as class that its called on is the target.

Reply:
Yeah, I think I prefer that.

Reply:
and done. looks better.